hooks/hardware-concurrency/index.js

```diff
@@ -14,22 +14,21 @@
  * limitations under the License.
  */
 
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 
-export const unsupportMessage = 'The Hardware Concurrency API is not supported on this platform.';
+export const UNSUPPORT_MESSAGE = 'The Hardware Concurrency API is not supported on this platform.';
```

Comment:
Similar to my comment on the other file, can we centralize the exports at the end of the file along with `export { useHardwareConcurrency };`?

Reply:
Yes for sure.